bootstrap/app.php

```diff
@@ -14,7 +15,9 @@
         apiPrefix: ''
     )
     ->withMiddleware(function (Middleware $middleware) {
-        //
+        $middleware->use([
+            CheckJamMiddleware::class,
+        ]);
```

Comment:
Am I incorrect in reading this as replacing the entire Laravel global middleware stack with this single class? I believe this to be the case because the `.../jam//playlist` route does not have the `CheckJamMiddleware` explicitly applied but still performs this check.

```suggestion
        //
```

I think this should be removed, and the middleware should be applied to each route (and route group in the near future.)

---

Additionally, routes such a `/me` and `/v1/jam/start/{playlist}` should not require a Jam to be in session.